index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+          .felx-container
+    {
+     display : flex;
+     justify-content: space-around;
+           
+    } 
+    
+    @media only screen and (max-width: 600px) {
+    body 
+    {
+    background-color: lightblue; 
+    }
+   
+  }
```

Comment:
Kindly In index.html from lines no 4 to 18 it's not a good practice to write CSS inside the HTML file, kindly avoid this.

Reply:
- [ ] Awesome!!! You can still explain further the reason(s) why it is not good to have CSS script in HTML file. Perhaps, it makes the code looks messy or something. 😄